src/server/server.js

```diff
@@ -5,7 +5,7 @@ import cors from 'cors';
 import bodyParser from 'body-parser';
 import jwt from 'express-jwt';
 import favicon from 'serve-favicon';
-import raven from 'raven';
+import Raven from 'raven';
```

Comment:
Wait. It was a capitalization problem?? Ugh.

Reply:
nawh, we bumped up a version & this is how they did it in the docs. I'm assuming the capitalize because it's a singleton? i dunno, ES2015 has straight up bastardized the when-to-capitalize rules